### DISCUSSION TITLE
Fit Training and Inferencing on a given class

### DISCUSSION THREAD

**Discussion by ghost** [autogluon/autogluon#1484] [Q&A]
Hi, is it possible to train a detector on a specific class (cat and dog) given a dataset containing multiple classes (e.g car, cat, dog, fishes).
Same question on inferencing.
Not sure If I'm clear.

**Comment by willsmithorg** [comment]
Are you trying to do tabular detection (tables of text/numeric data) or image detection?  If image, AutoGluon can identify multiple classes in a single image : see [https://auto.gluon.ai/stable/tutorials/object_detection/index.html ](https://auto.gluon.ai/stable/tutorials/object_detection/index.html)

**Reply by ghost** [reply]
Okay, my post wasn't clear sorry, let me explain it better : 
I have a dataset with multiple classes (let say dog, cats and chicken) and I want to train an Image Detection on ONLY one class (chicken), Is there an option to specify to the trainer process to work only on the chicken in my dataset (and I can't separate pictures because I have chicken and dog on the same pictures). 
Hope that's help to understand.
